### DISCUSSION TITLE
will Deployer 7 eventually support Symfony 6?

### DISCUSSION THREAD

**Discussion by arderyp** [deployphp/deployer#3245] [General]
Or will this be pushed off into a new major Deploy 8 version?

**Comment by antonmedv** [comment]
I doubt it. Use v7.

**Reply by arderyp** [reply]
@antonmedv I am using 7. I am asking about 7.  7 is currently bound to Symfony 5 via composer.  I am using Symfony 5.4 and deployer 7.  I hope to update to Symfony 6.4 early next year and I am wondering if Deployer is going to hold me back, or whether Deployer v7 will eventually become compatible with Symfony 6.4

**Reply by antonmedv** [reply]
No, as Deployer distributed only as phar. Even via composer. With no deps.

**Reply by arderyp** [reply]
Ok. But it's included in my PHP project now and includes source, not just a phar file.  That was reverted in the last RC and maintained in 7.0.0

**Reply by antonmedv** [reply]
Source is added, but no deps.

**Reply by arderyp** [reply]
Okay, maybe I'm just misunderstanding the infrastructure.  So you think it could be included on symfony 6 projects now without conflict? If so, that's awesome news!

**Reply by antonmedv** [reply]
Yes, give it a try.

**Reply by svnldwg** [reply]
@arderyp did you manage to make it work? if so i would be interested in your solution. we are having the same issues, that deployer/deployer v7 is not compatible with our symfony 6 application.

**Reply by antonmedv** [reply]
What do you mean not compatible?

**Reply by svnldwg** [reply]
I mean this:

> Problem 1
    - deployer/deployer v7.0.0-rc.2 requires symfony/console ^5 -> found symfony/console[v5.0.0-BETA1, ..., 5.4.x-dev] but it conflicts with your root composer.json require (^v6.1.4).

This makes it impossible to install `deployer/deployer` as a composer dependency when using symfony 6, because `deployer/deployer` requires symfony 5: https://github.com/deployphp/deployer/blob/master/composer.json#L49

https://deployer.org/docs/6.x/installation#source-composer-installation

**Reply by antonmedv** [reply]
Try 7.0.1. It was fixed.

**Reply by arderyp** [reply]
I haven't tried yet, but psyched you all got it fixed/working.  Hopefully I'll be following suit shortly.
